@kiva/kv-components/vue/KvContentfulImg.vue

```diff
@@ -86,7 +91,6 @@ export default {
 		* */
 		fallbackFormat: {
 			type: String,
-			required: true,
```

Comment:
Since this prop has a default it doesn't need to be required

Reply:
Is that field needed to activate the validator?

Reply:
no it doesn't appear to be.